centreon/www/class/centreonWidget/Params/Connector/Service.class.php

```diff
@@ -103,7 +103,9 @@ protected function getServiceIds($hostId)
         $res = $this->db->query($sql);
         $tab = array();
         while ($row = $res->fetchRow()) {
-            $tab[$hostId . "-" . $row['service_id']] = $row['service_description'];
+            # For meta services, use display_name column instead of service_description
```

Comment:
The `#` character is unconventional for comments in 2023, except for specific Attribute case.
Please use `//`